### PR TITLE
Play quiz page

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review the quiz page of my quiz site

It has the following features

nav : links to categories and rules page
main: it has the question box with the question number, score, question, answer, and also the correct / wrong answer marked with a different colors.
Footer: links to the social websites

image

image

### CODE REVIEW

File: play-quiz/play.css [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+    font-family: 'Inter', sans-serif;
+    font-family: 'Mukta', sans-serif;
+    background-image: url("../images/blue-copy-space-digital-background_23-2148821698.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+   
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+/* background-color: #e056fd;
```
Comment (pnchinmay) [inline play-quiz/play.css]: Remove these comments

File: play-quiz/play.css [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    color: var(--hover-color);
+    border-bottom: 1px solid var(--hover-color);
+  
+}
+
+
+main{
+    display: flex;
+    flex-direction: column;
+
```
Comment (pnchinmay) [inline play-quiz/play.css]: Remove these comments

**Review by pnchinmay — APPROVED** [review]
Left a few comments
Everything else looks good :100:

File: play-quiz/play.css [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+nav h2{
+    position: relative;
+  
+    font-size: 4rem;
+    line-height: 1.2rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Inter', sans-serif;
```
Comment (khushijohri2001) [inline play-quiz/play.css]: Keep the indentation same everywhere. Ideally, 2 spaces in HTML/CSS according to Google Style Guide

**Review by khushijohri2001 — APPROVED** [review]

**Pull request merged**